raft/raft.go

```diff
@@ -261,15 +307,19 @@ type raft struct {
 	// isLearner is true if the local raft node is a learner.
 	isLearner bool
 
+	// msgs contains the list of messages that should be sent out immediately to
+	// other nodes.
+	//
+	// Messages in this list must target other nodes.
 	msgs []pb.Message
-
-	// voteSelfOnAdvance is a marker that the local raft node should vote for
-	// itself upon its next call to advance. This is not meant to be the final
-	// approach towards handling self-votes, but it's a useful intermediate
-	// point to get all tests working and to write some additional tests that
-	// demonstrate possible race conditions when self-voting is asynchronous.
-	// This is replaced in a later commit.
-	voteSelfOnAdvance pb.Message
+	// msgsAfterAppend contains the list of messages that should be sent after
+	// the accumulated unstable state (e.g. term, vote, []entry, and snapshot)
+	// has been persisted to durable storage. This includes waiting for any
+	// unstable state that is already in the process of being persisted (i.e.
+	// has already been handed out in a prior Ready struct) to complete.
+	//
+	// Messages in this list may target other nodes or may target this node.
```

Comment:
may target other nodes (MsgAppResp, MsgVoteResp) or may target this node (<fill in examples; MsgAppResp too?>, MsgStorageAppendResp)

Reply:
Done.